bundles/AdminBundle/Resources/public/js/pimcore/document/edit.js

```diff
@@ -404,6 +404,34 @@ pimcore.document.edit = Class.create({
         catch (e2) {
         }
 
+        return values;
+    },
+
+    getRequiredEditableValues: function () {
```

Comment:
In my opinion we should call this method `getEmptyRequiredEditables` as this is what this function is actually doing (it is not returning the values of the required editables).